hudi-hive/src/main/java/org/apache/hudi/hive/util/SchemaUtil.java

```diff
@@ -402,7 +402,7 @@ public static String generateCreateDDL(MessageType storageSchema, HiveSyncConfig
 
     String partitionsStr = partitionFields.stream().collect(Collectors.joining(","));
     StringBuilder sb = new StringBuilder("CREATE EXTERNAL TABLE  IF NOT EXISTS ");
-    sb = sb.append(config.databaseName).append(".").append(config.tableName);
+    sb = sb.append("`").append(config.databaseName).append("`.`").append(config.tableName).append("`");
```

Comment:
Could we define \` as a viarable, and use it in HoodieHiveClient?

Reply:
Moved it to a variable